src/frontend/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+//import "../App.css";
```

Comment:
If we do not need to CSS file to be imported, can we delete the line instead of just commenting it out?

Reply:
@Selchuck will delete app.css